ui/api/src/main/java/org/jboss/forge/addon/ui/input/SelectComponent.java

```diff
@@ -30,4 +31,9 @@
    Converter<VALUETYPE, String> getItemLabelConverter();
 
    IMPLTYPE setItemLabelConverter(Converter<VALUETYPE, String> converter);
+
+   Converter<VALUETYPE, String> getInteractionItemLabelConverter();
+
```

Comment:
Hey @gastaldi, I saw others methods and I found this: https://github.com/forge/core/blob/master/javaee/impl/src/main/java/org/jboss/forge/addon/javaee/ejb/ui/EJBSetMethodTransactionAttributeCommand.java

This case the convert is applied in both case. (interaction presentation and no interaction presentation)

I think that "Maybe we should change the UIContext.getProvider().isGUI() to return true when interaction mode is enabled" not is the better idea.

Maybe the convert should be applied always. 
For NewProject, the convert is applied only in GUI, because is a rule of it.

This case, we'll not break some existent scripts with this command, because the shell presentation will not have changes.

So, I'll add the getExtension in ProjectType and change the convert for this input in NewProjectWinzard. WDYT?